### PR TITLE
Adding Australis config files to aurora and hw2

### PR DESCRIPTION
**Pull request by nsai1**
Signed-off-by: Naian Sai <naian.sai@intel.com>

### CODE REVIEW

**Review by avincigu — COMMENTED** [review]
Are those two files supposed to be empty?

**Review by avincigu — APPROVED** [review]

File: configurations/hw2/MachineConfig_Australis.json [hunk 1]
```diff
@@ -0,0 +1,279 @@
+{
+  "UcsConfigValues": [
```
Comment (sindhu-106) [inline configurations/hw2/MachineConfig_Australis.json:2]: I think hw2 folder name is used for different names before aurora. Are these changes necessary for hw2 folder.

File: configurations/hw2/SystemManifest_Australis.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
+{
+  "sysname": "Australis",
+  "views": {
+    "Full": {
+      "view": "Full",
+      "view-description": "Full Floor Layout",
+      "initzoom": 0,
+      "zoomscales": [4, 6, 8],
+      "rackscale": 12,
+      "floor": {
```
Comment (sindhu-106) [inline configurations/hw2/SystemManifest_Australis.json:10]: same here.

File: configurations/hw2/MachineConfig_Australis.json (hunk 1 above)
Comment (nsai1) [reply configurations/hw2/MachineConfig_Australis.json:2]: Yes they are, also they serve as a back up for the files in Aurora incase the ones in that folder get overwritten by another script. Adding these files to this HW2 folder was made in the GUI meeting.

**Review by sindhu-106 — APPROVED** [review]
